src/edu/stanford/dlss/wowza/SulWowza.java

```diff
@@ -364,16 +369,55 @@ HttpURLConnection getStacksHttpURLConn(URL stacksUrl, String requestMethod)
         return stacksConn;
     }
 
-    static String urlEncode(String urlComponent)
+    /** We define these escape methods, because the most obvious method to use (URLEncoder.encode) works fine
```

Comment:
BTW,  while I appreciate your attention to detail, my reaction to these big comment blocks is TL;DR.    My version would be something like

"We need to do xxx according to these specs;  URLEncoder.encode  does the following incorrectly: xxxx;  Guava makes it possible for us to correctly distinguish between form/query params encoding and URL encoding (for the druid in the path)."